lib/nostrum/api.ex

```diff
@@ -557,25 +557,33 @@ defmodule Nostrum.Api do
 
   If successful, returns `{:ok, users}`. Otherwise, returns `t:Nostrum.Api.error/0`.
 
+  The optional `params` are `after`, the user ID to query after, absent by default,
+  and `limit`, the max number of users to return, 1-100, 25 by default.
+
   See `create_reaction/3` for similar examples.
   """
-  @spec get_reactions(Channel.id(), Message.id(), emoji) :: error | {:ok, [User.t()]}
-  def get_reactions(channel_id, message_id, emoji)
+  @spec get_reactions(Channel.id(), Message.id(), emoji, keyword()) :: error | {:ok, [User.t()]}
+  def get_reactions(channel_id, message_id, emoji, params \\ [])
 
-  def get_reactions(channel_id, message_id, %Emoji{} = emoji),
-    do: get_reactions(channel_id, message_id, Emoji.api_name(emoji))
+  def get_reactions(channel_id, message_id, %Emoji{} = emoji, params),
+    do: get_reactions(channel_id, message_id, Emoji.api_name(emoji), params)
 
-  def get_reactions(channel_id, message_id, emoji_api_name) do
-    request(:get, Constants.channel_reactions_get(channel_id, message_id, emoji_api_name))
+  def get_reactions(channel_id, message_id, emoji_api_name, params) do
+    request(
+      :get,
+      Constants.channel_reactions_get(channel_id, message_id, emoji_api_name),
+      "",
+      params
+    )
     |> handle_request_with_decode({:list, {:struct, User}})
   end
 
   @doc ~S"""
-  Same as `get_reactions/3`, but raises `Nostrum.Error.ApiError` in case of failure.
+  Same as `get_reactions/4`, but raises `Nostrum.Error.ApiError` in case of failure.
```

Comment:
This should be
```suggestion
  Same as `get_reactions/3-4`, but raises `Nostrum.Error.ApiError` in case of failure.
```
right?

Reply:
no, that breaks linking to the other function

Reply:
![image](https://user-images.githubusercontent.com/38387397/226119143-1b689874-b4c6-4786-80db-b9168fd12b6f.png)
Legally the functions are arity 4